Makefile

```diff
@@ -172,9 +172,16 @@ bin/Test$(CONFIGURATION)/$(JAVA_INTEROP_LIB): bin/$(CONFIGURATION)/$(JAVA_INTERO
 run-android: $(ATESTS)
 	(cd src/Android.Interop/Tests; $(MSBUILD) $(MSBUILD_FLAGS) '/t:Install;RunTests' $(if $(FIXTURE),/p:TestFixture=$(FIXTURE)))
 
-run-test-jnimarshal: bin/Test$(CONFIGURATION)/Java.Interop.Export-Tests.dll bin/Test$(CONFIGURATION)/$(JAVA_INTEROP_LIB)
+JRE_DLL_CONFIG=bin/$(CONFIGURATION)/Java.Runtime.Environment.dll.config
+
+$(JRE_DLL_CONFIG): src/Java.Runtime.Environment/Java.Runtime.Environment.csproj
+	$(MSBUILD) $(MSBUILD_FLAGS) $<
+
+run-test-jnimarshal: bin/Test$(CONFIGURATION)/Java.Interop.Export-Tests.dll bin/Test$(CONFIGURATION)/$(JAVA_INTEROP_LIB) $(JRE_DLL_CONFIG)
+	mv -fv "$(JRE_DLL_CONFIG)" "$(JRE_DLL_CONFIG).bak"
```

Comment:
I still don't understand why this line and the associated `mv -fv` below are required, and [your earlier reply doesn't help](https://github.com/xamarin/java.interop/pull/334#discussion_r196438324).

Nothing in this recipe creates that file -- and I don't see anything *within* this recipe which would create the file! -- why would it need to be backed up and restored?

Alternatively, perhaps the `jnimarshalmethod-gen.exe` invocation *fails* if `Java.Runtime.Envonment.dll.config` exists?  In which case that sounds deeply problematic?

I feel like there's something I don't understand here, and I'd really like to understand it.